l10n_it_fatturapa_out_dn/wizard/wizard_export_fatturapa.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Marco Colommbo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    @api.model
+    def default_get(self, fields):
+        res = super(WizardExportFatturapa, self).default_get(fields)
+        invoice_ids = self.env.context.get("active_ids", False)
+        invoices = self.env["account.move"].browse(invoice_ids)
+        for invoice in invoices:
+            for line in invoice.invoice_line_ids:
+                if line.delivery_note_id:
+                    res["include_transport_data"] = "dati_dn"
+                    return res
```

Comment:
è corretto fare `return` qui?

Reply:
Il default per include_transport_data diventa dati_dn se è presente anche una sola riga relativa a un dn in una qualsiasi fattura. Probabilmente c'è un modo più compatto tipo
if any(invoices.invoices_line_ids.mapped("delivery_note_id"))?

Reply:
sì, direi che la soluzione che proponi è decisamente più pythonica, hai voglia di modificare questa parte di codice?

Reply:
Fatto